lab-phelan/test/arithmetic.test.js

```diff
@@ -0,0 +1,17 @@
+const arith = require(`../lib/arithmetic`);
+
+test('Testing arithmetic > add function [positive case].', () => {
+  expect(arith.add(1847, 948372)).toBe(950219);
+});
+
+test('Testing arithmetic > add function [failure case].', () => {
+  expect(arith.add(1847, 'wrong')).toBe(null);
+});
+
+test('Testing arithmetic > sub function [positive case].', () => {
```

Comment:
to be more descriptive you could have a describe block that says( 'testing arithmetic sub method') then test('it should return the product of a - b')